client/point.lua

```diff
@@ -59,14 +59,14 @@ RegisterCommand('point', function()
 
             local coords = GetOffsetFromEntityInWorldCoords(ped, (cosCamHeading * -0.2) - (sinCamHeading * (0.4 * camHeading + 0.3)), (sinCamHeading * -0.2) + (cosCamHeading * (0.4 * camHeading + 0.3)), 0.6)
             local ray = Cast_3dRayPointToPoint(coords.x, coords.y, coords.z - 0.2, coords.x, coords.y, coords.z + 0.2, 0.4, 95, ped, 7);
-            nn,blocked,coords,coords = GetRaycastResult(ray)
-
-            Citizen.InvokeNative(0xD5BB4025AE449A4E, ped, "Pitch", camPitch)
-            Citizen.InvokeNative(0xD5BB4025AE449A4E, ped, "Heading", camHeading * -1.0 + 1.0)
-            Citizen.InvokeNative(0xB0A6CFD2C69C1088, ped, "isBlocked", blocked)
-            Citizen.InvokeNative(0xB0A6CFD2C69C1088, ped, "isFirstPerson", Citizen.InvokeNative(0xEE778F8C7E1142E2, Citizen.InvokeNative(0x19CAFA3C87F7C2FF)) == 4)
+            nn,blocked,coords,coords = GetRaycastResult(ray)	
+			SetTaskMoveNetworkSignalFloat (ped, "Pitch", camPitch)
+			SetTaskMoveNetworkSignalFloat (ped, "Heading", camHeading * -1.0 + 1.0)
+			SetTaskMoveNetworkSignalBool  (ped, "isBlocked", blocked)
+			SetTaskMoveNetworkSignalBool  (ped, "isFirstPerson",  false)
```

Comment:
Remove the spaces between the function and the parameters 

Also why did you remove the last part of the last native there replacing it with false?
`Citizen.InvokeNative(0xB0A6CFD2C69C1088, ped, "isFirstPerson", Citizen.InvokeNative(0xEE778F8C7E1142E2, Citizen.InvokeNative(0x19CAFA3C87F7C2FF)) == 4)`

to keep it 1:1 just replacing the natives it would be 
```lua
SetTaskMoveNetworkSignalBool(ped, "isFirstPerson", GetCamViewModeForContext(GetCamActiveViewModeContext()) == 4)
```
Though I'm wondering what's up with that logic in general as the last 2 natives return a number but the main native wants a boolean

Reply:
This was done
Thanks you @zfbx